### PR TITLE
Fix for issue #53: wrong source file path returned under Windows.

### PR DESCRIPTION
**Pull request by vcouvert**
Hello,

Thanks for reviewing this modification.

Vincent

### CODE REVIEW

File: src/MICmnLLDBDebugSessionInfo.cpp [hunk 1]
```diff
@@ -306,6 +318,17 @@ bool CMICmnLLDBDebugSessionInfo::ResolvePath(const CMIUtilString &vstrUnknown,
   // No files exist in the union of working directory and debuginfo path
   // Simply use the debuginfo path and let the IDE handle it.
 
+#ifdef _WIN32 // Under Windows we must replace "\\" by "/"
+  CMIUtilString strTestPath = vecPathFolders[0]; // Drive letter
+  MIuint nFolders = 1;
+  while (vecPathFolders.size() > nFolders) {
+    strTestPath += "/";
```
Comment (tkrasnukha) [inline src/MICmnLLDBDebugSessionInfo.cpp:326]: I suggest creating a constant that would by `"\\"` for Windows and `"/"` for others. Then you will be able just to replace literals with it in statements `vwrResolvedPath.Split("/", vecPathFolders);` and `strTestPath += "/";`

File: src/MICmnLLDBDebugSessionInfo.cpp [hunk 2]
```diff
@@ -306,6 +323,17 @@ bool CMICmnLLDBDebugSessionInfo::ResolvePath(const CMIUtilString &vstrUnknown,
   // No files exist in the union of working directory and debuginfo path
   // Simply use the debuginfo path and let the IDE handle it.
 
+#ifdef _WIN32 // Under Windows we must replace "\\" by "/"
+  CMIUtilString strTestPath = vecPathFolders[0]; // Drive letter
```
Comment (tkrasnukha) [inline src/MICmnLLDBDebugSessionInfo.cpp:327]: `strTestPath` already has the correct value after the last iteration of the loop above. If you move its declaration out of the first loop and clear `strTestPath` at the beginning of every iteration, you will only need to assign its value to `vwrResolvedPath`  here.

Comment (vcouvert) [reply src/MICmnLLDBDebugSessionInfo.cpp:327]: You are right I will upadte my changes.

**Comment by vcouvert** [comment]
I have merge/squash issues on this PR, I will do an other one.

**Pull request closed without merging**